src/types.ts

```diff
@@ -210,7 +210,7 @@ export interface ManifestOptions {
     short_name?: string
     url: string
     description?: string
-    icons: string[]
+    icons: Record<string, any>[]
```

Comment:
Should support both type and do a normalization in runtime

Reply:
from 1 string to this https://www.w3.org/TR/image-resource/#processing-an-image-resource-from-json? we need to add some defaults for `sizes` (`96x96`) and `type` (`image/png`) and we can omit `purpose`: what about more than 1 icon? 

Maybe it would be better adding the string type union and throw error or show warning when using string.